readme.txt

```diff
@@ -1,9 +1,9 @@
 === Monero WooCommerce Extension ===
-Contributors: serhack
+Contributors: serhack, mosu-forge
```

Comment:
?

Reply:
If this PR is merged, then he is indeed a contributor and this should be here.